tests/test_app.py

```diff
@@ -576,6 +576,21 @@ def test_falsy_param(app):
     response = json.loads(resp.data.decode())
     assert response == 1
 
+
+def test_formData_param(app):
+    app_client = app.app.test_client()
+    resp = app_client.post('/v1.0/test-formData-param', data=dict(formData='test'))
```

Comment:
Thanks for the PR. Can you just change the dict here and on line 590 to the literal form (`{formData: ....}`) for consistency with the rest of the tests?